packages/lesswrong/components/linkPreview/PostLinkPreview.tsx

```diff
@@ -624,11 +624,66 @@ const manifoldStyles = (theme: ThemeType): JssStyles => ({
     width: 560,
     height: 405,
     border: "none",
-    maxWidth: "100vw",
+    maxWidth: "100vw"
```

Comment:
Unimportant, but I'm confused why removing this comma was important enough to get its own commit

Reply:
It happened when I changed some styling back and forth, and then left in (what seems to me) the nicer-looking way. I didn't think much about it. 

I think I have some disagreement with the rest of y'all (LW and EAF both) on going ahead and including things like this? (similar to slight whitespace changes that make it easier to read, although to be fair often the whitespace changes are just an accident). 

IMO it's not worth spending the conversation-overhead on things like this unless they look actively wrong.